native/src/components/NothingFound.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
+import styled from 'styled-components/native'
+
+import SadIcon from '../assets/smile-sad.svg'
```

Comment:
I think this could/should be done in a separate issue. E.g. together with https://issues.tuerantuer.org/browse/IGAPP-189 or probably better with https://issues.tuerantuer.org/browse/IGAPP-921. Do you want to add your proposal to that issue? I will adjust the naming here.

Reply:
I will add it to IGAPP-189.